Bug: Release builds not working

Building in release currently generates an error. This also prevents building the installer by adding CONFIG+=installer to the qmake build instructions.

Tested with commit edb4b67cae73c5a9529e5722a33feb5702c2fd8d.
Clean rebuild.
Results in:
\src\comm\MockLink.h:73: error: C2039: 'TypeMock': is not a member of 'LinkConfiguration'
\src\comm\MockLink.h:73: error: C2065: 'TypeMock': undeclared identifier

--- a/qgroundcontrol.pro
+++ b/qgroundcontrol.pro
@@ -456,8 +456,7 @@ HEADERS += \
     src/api/QGCOptions.h \
     src/api/QGCSettings.h \
     src/api/QmlComponentInfo.h \
-    src/GPS/Drivers/src/base_station.h \
-    src/qgcunittest/NetCdfTest.h
+    src/GPS/Drivers/src/base_station.h
 
 contains (DEFINES, QGC_ENABLE_PAIRING) {
     HEADERS += \
@@ -475,8 +474,7 @@ SOURCES += \
     src/api/QGCCorePlugin.cc \
     src/api/QGCOptions.cc \
     src/api/QGCSettings.cc \
-    src/api/QmlComponentInfo.cc \
-    src/qgcunittest/NetCdfTest.cc
+    src/api/QmlComponentInfo.cc
 
 contains (DEFINES, QGC_ENABLE_PAIRING) {
     SOURCES += \
@@ -541,6 +539,7 @@ DebugBuild { PX4FirmwarePlugin { PX4FirmwarePluginFactory { APMFirmwarePlugin { 
         #src/qgcunittest/FileManagerTest.h \
         #src/qgcunittest/MainWindowTest.h \
         #src/qgcunittest/MessageBoxTest.h \
+        src/qgcunittest/NetCdfTest.h
 
     SOURCES += \
         src/Audio/AudioOutputTest.cc \
@@ -591,6 +590,7 @@ DebugBuild { PX4FirmwarePlugin { PX4FirmwarePluginFactory { APMFirmwarePlugin { 
         #src/qgcunittest/FileManagerTest.cc \
         #src/qgcunittest/MainWindowTest.cc \
         #src/qgcunittest/MessageBoxTest.cc \
+        src/qgcunittest/NetCdfTest.cc
 
 } } } } } }
 

